tests/core/asyncio/test_broadcast_config.py

```diff
@@ -18,7 +18,9 @@ class BroadcastEvent(BaseEvent):
     client_a.subscribe(BroadcastEvent, return_queue.put_nowait)
     client_b.subscribe(BroadcastEvent, return_queue.put_nowait)
 
-    await server.wait_until_all_remotes_subscribed_to(BroadcastEvent)
+    await server.wait_until_remote_subscribed_to(client_a.name, BroadcastEvent)
+    await server.wait_until_remote_subscribed_to(client_b.name, BroadcastEvent)
```

Comment:
Oh and I think we should really do the rename`wait_until_remote_subscribed_to` -> `wait_until_endpoint_subscribed_to`. The name `remote` is a bit misleading here.